docs/interpreter/flink.md

```diff
@@ -166,15 +171,39 @@ Zeppelin will create 4 variables to represent flink's entrypoint:
 * `stenv`   (StreamTableEnvironment) 
 * `btenv`   (BatchTableEnvironment)
 
+## Flink Planner
+
+Starting from Flink 1.9, there're 2 planners supported by Flink's table api: flink & blink.
+* If you want to use DataSet api, then please use flink planner (specify `zeppelin.flink.planner` to `flink`).
+* In other cases, we would always recommend you to use blink planner which is also the default value of `zeppelin.flink.planner`.
+
+## How to use Hive
+
+In order to use Hive in Flink, you have to do several setting.
+* Set `zeppelin.flink.enableHive` to `true`
+* Copy necessary dependencies to flink's lib folder, check this [link](https://ci.apache.org/projects/flink/flink-docs-release-1.9/dev/table/hive/#depedencies) for more details 
+  * flink-connector-hive_{scala_version}-{flink.version}.jar
+  * flink-hadoop-compatibility_{scala_version}-{flink.version}.jar
+  * flink-shaded-hadoop-2-uber-{hadoop.version}-{flink-shaded.version}.jar
+  * hive-exec-2.x.jar (for Hive 1.x, you need to copy hive-exec-1.x.jar, hive-metastore-1.x.jar, libfb303-0.9.2.jar and libthrift-0.9.2.jar)
```

Comment:
should we use verbatim for file names, like, it's done for `zeppelin-env.sh` ?

Reply:
Do you mean to specify them in `zeppelin-env.sh` ?

Reply:
No, I mean to use monospace for them, like `file_name`

Reply:
These jar file names is named by upstream project (flink/hive), we can not change them.